### PR TITLE
feat: added pydantic for data validation

### LINKED ISSUE #38: feat: add pydantic for data validation
Use pydantic to validate request data before adding to the database

### PR DESCRIPTION
**Pull request by karelumair**
Added pydantic to validate the request body.

closes #38

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -22,9 +22,11 @@ html5lib = "^1.1"
 flask-expects-json = "^1.7.0"
 celery = "^5.2.7"
 redis = "^4.5.1"
+pydantic = "^1.10.4"
+flask-pydantic = "^0.11.0"
```
Comment (pratikGhodke1) [inline pyproject.toml]: Do we need to install both differently?

Comment (karelumair) [reply pyproject.toml]: No, actually I forgot to remove it. will do that.

Comment (karelumair) [reply pyproject.toml]: Done.

**Review by pratikGhodke1 — APPROVED** [review]
Looks Good! We might need to do refactoring of code and folder structure afterwards. But for now  we are good!

**Pull request merged**